### DISCUSSION TITLE
File Submission Deferment - Rx Drug Information

### DISCUSSION THREAD

**Discussion by djohnson071815** [CMSgov/price-transparency-guide#479] [Q&A] [answered]
It is our understanding that the Department has deferred enforcement on the machine-readable files related to prescription drug pricing while it considers, through notice-and-comment rulemaking, whether the prescription drug machine-readable file requirement remains appropriate.

What is not clear is if this deferred enforcement is applicable only to the one file specific to prescription drug pricing. Or, does the deferral apply to prescription drug information in all 3 files (including the "allowed amounts" and "in-network rate" files)?

**Comment by shaselton-usds** [comment, marked as answer]
The deferment only applies to the prescription drug file. All other drug disclosure requirements for the other two files are still required.
